device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttTest.java

```diff
@@ -1253,8 +1253,8 @@ public void receiveSuccessCustomPropertyHasUnusualCharacters() throws TransportE
         assertEquals("$", receivedMessage.getProperties()[0].getValue());
         assertEquals("&", receivedMessage.getProperties()[1].getValue());
         assertEquals("%", receivedMessage.getProperties()[2].getName());
-        assertEquals("\"", receivedMessage.getProperties()[2].getValue());
```

Comment:
This test was bad because these characters should never appear in an application property